### DISCUSSION TITLE
Is Age of Mythology support planned for openage?

### DISCUSSION THREAD

**Discussion by aredfan** [SFTtech/openage#1555] [Q&A] [answered]
Hi. I've been wondering for a while whether there's plans for openage to support Age of Mythology.

**Comment by Mr-Bajs** [comment, marked as answer]
Age of Empires 2 is 2D and Age of myhology is 3D so i would assume no.

**Reply by heinezen** [reply]
This is correct. Although the renderer is capable of rendering in 3D, it is very unlikely that we add AoM because it has different requirements.

Maybe someone can do a demake of the AoM assets to 2D. Then we could likely support them outright :D

**Reply by aredfan** [reply]
Ah I see, my mistake.  It's been many years since I played AoE or AoM, so I didn't realise the games have different graphics and requirements.  Thank you for explaining 👍
